### PR TITLE
bsp: imx: add apalis-imx6 config

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
This configuration supports the apalis-imx6 Toradex SoM.

Signed-off-by: Tyler Baker <tyler@foundries.io>

### CODE REVIEW

File: bsp/imx/apalis-imx6.cfg [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+CONFIG_DRM_DW_HDMI_CEC is not set
+CONFIG_DRM_DW_HDMI_AHB_AUDIO is not set
+CONFIG_DRM_IMX is not set
+CONFIG_DRM_IMX_PARALLEL_DISPLAY is not set
+CONFIG_DRM_IMX_TVE is not set
+CONFIG_DRM_IMX_LDB is not set
+CONFIG_DRM_IMX_HDMI is not set
+CONFIG_DRM_ETNAVIV is not set
+CONFIG_DRM_ETNAVIV_THERMAL is not set
+CONFIG_DRM_PANEL_ORIENTATION_QUIRKS is not set
```
Comment (ricardosalveti) [inline bsp/imx/apalis-imx6.cfg]: Seems to be missing "# " at the beginning of every config that should be disabled. Did you check the final config to see if they were indeed disabled here?

Comment (EmbeddedAndroid) [reply bsp/imx/apalis-imx6.cfg]: So it did indeed disable them, but I can add the '#' for consistency sake.

Comment (ricardosalveti) [reply bsp/imx/apalis-imx6.cfg]: Please, just to get it aligned with the other fragments.

Comment (EmbeddedAndroid) [reply bsp/imx/apalis-imx6.cfg]: Ok, this should be done now in the latest commit.

**Comment by ricardosalveti** [comment]
LGTM

**Pull request merged**